h/static/scripts/directive/thread.coffee

```diff
@@ -210,6 +215,10 @@ module.exports = [
         render ->
           ctrl.container = thread
           scope.$digest()
+          if (ctrl.id and not ctrl.container?.message?.id)
+            # Scroll the sidebar to show new annotations.
+            $location.hash(ctrl.id)
+            $anchorScroll()
```

Comment:
Funny to see another name collision arising; this time for anchor.
(True, given the existing terms used in HTML world, it always lingered around, but at least we didn't really use that term inside our code, up now now.)

That is not to say that this is wrong: we could rename the other `anchors`, as discussed earlier. (Maybe to `TargetLocation`, or something like that.)